drivers/nrf_802154/sl/include/platform/nrf_802154_irq.h

```diff
@@ -80,7 +80,7 @@ typedef void (* nrf_802154_isr_t)();
  * @param[in] prio  Priority of the IRQ.
```

Comment:
The docs could reflect what a negative value means.

Reply:
The meaning of `prio` is completely up to the platform, `prio` does not go directly to NVIC (however it can, id does so in our bare metal platform), as Zephyr can shift it. Also the way negative values are interpreted are up to platform. Zephyr is one of the platforms. However this file is platform-independent, but passes platform-dependent `prio` value.
What shall be written here?

Reply:
@ankuns maybe it'd be enough to state the below:
```c
* @param[in] prio Priority of the IRQ.
* @param[in] isr  Pointer to ISR.
*
* @note Interpretation of the value represented by @p prio is platform-dependent and defined by
*       the implementation. 
```

Reply:
done